src/main/webapp/app/trial/trial.component.html

```diff
@@ -38,6 +38,8 @@
             <span id="nctId" style="margin-left:10px">
                 <a target="_blank" href="https://clinicaltrials.gov/ct2/show/{{trialChosen.nct_id}}">{{ trialChosen.nct_id }}</a>
             </span>
+            <button class="btn btn-primary" (click)="loadMongo()" style="margin-left: 10px;">Finish Curation</button>
```

Comment:
This section should only be visible in production mode.

Reply:
Thanks for pointing out.